queue_services/entity-bn/requirements.txt

```diff
@@ -18,6 +18,8 @@ sentry-sdk==1.20.0
 six==1.15.0
 urllib3==1.26.11
 git+https://github.com/bcgov/business-schemas.git@2.15.21#egg=registry_schemas
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+#git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+# fetch legal api's updates from the feature-db-versioning branch, need to change to the main branch later
+git+https://github.com/bcgov/lear.git@feature-db-versioning#egg=legal_api&subdirectory=legal-api
```

Comment:
Now, we'll pull legal-api's update from the feature-db-versioning branch, and switch back to the main later